src/index.html

```diff
@@ -15,6 +15,7 @@
   <link rel="preload" href="titilliumweb-italic-webfont.woff2" as="font" type="font/woff2" crossorigin="anonymous">
   <link rel="preload" href="fa-solid-900.woff2" as="font" type="font/woff2" crossorigin="anonymous">
   <link rel="preload" href="primeicons.ttf" as="font" type="font/ttf" crossorigin="anonymous">
+  <script src="https://unpkg.com/@phosphor-icons/web"></script>
```

Comment:
Could you embed them in the project instead? And I don't think we need all variations. (can be done in a separate PR)

Reply:
Fixed #1362